MovieMVP/ListMovieModule/View/ListMoviesViewController.swift

```diff
@@ -249,32 +230,57 @@ final class ListMoviesViewController: UIViewController {
     private func createMainActivityIndicatorViewConstraint() {
         mainActivityIndicatorView.translatesAutoresizingMaskIntoConstraints = false
         NSLayoutConstraint.activate([
-            mainActivityIndicatorView.topAnchor.constraint(equalTo: view.layoutMarginsGuide.topAnchor, constant: -30),
+            mainActivityIndicatorView.topAnchor.constraint(
+                equalTo: view.layoutMarginsGuide.topAnchor,
+                constant: Constants.minusThirtyNumber
+            ),
             mainActivityIndicatorView.rightAnchor.constraint(equalTo: view.layoutMarginsGuide.rightAnchor)
         ])
     }
 }
 
-// MARK: - UITableViewDelegate, UITableViewDataSource
+// MARK: - UITableViewDataSource
 
-extension ListMoviesViewController: UITableViewDelegate, UITableViewDataSource {
+extension ListMoviesViewController: UITableViewDataSource {
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        movies?.count ?? 0
+        presenter.movies?.count ?? .zero
     }
 
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         guard
             let cell = tableView
             .dequeueReusableCell(withIdentifier: Constants.movieTableViewCellText) as? MovieTableViewCell,
-            let safeMovies = movies
+            let safeMovies = presenter.movies
         else { return UITableViewCell() }
-        cell.configureMovieTableViewCell(movie: safeMovies[indexPath.row])
+        presenter.currentMovie = safeMovies[indexPath.row]
+        cell.configureMovieTableViewCell(presenter: presenter)
```

Comment:
```suggestion
        cell.configureMovieTableViewCell(presenter: presenter)
```
я бы не передавал весь presenter, а достал нетворк

Reply:
У нас презентер ответственен за всю логику поэтому передаю именно его, что бы не вырывать части презентера и не раскидывать их по приложению, тогда логика презентера теряется